src/backend/src/controllers/design-review.controllers.ts

```diff
@@ -24,6 +24,43 @@ export default class DesignReviewController {
     }
   }
 
+  static async createDesignReview(req: Request, res: Response, next: NextFunction) {
+    try {
+      const submitter: User = await getCurrentUser(res);
+      const {
+        dateScheduled,
+        teamTypeName,
+        requiredMembers,
```

Comment:
this should be memberIds cause you changed it in the route

Reply:
make sure all of these are updated to match the route (e.g. teamTypeName should be teamTypeId)